Source/WebCore/rendering/svg/SVGResources.cpp

```diff
@@ -213,19 +213,25 @@ bool SVGResources::buildCachedResources(const RenderElement& renderer, const Ren
 
     const AtomString& tagName = element.localName();
     if (tagName.isNull())
-        return false;
+        return nullptr;
 
     const SVGRenderStyle& svgStyle = style.svgStyle();
 
-    bool foundResources = false;
+    auto ensureResources = [](std::unique_ptr<SVGResources>& resources) -> std::unique_ptr<SVGResources>& {
```

Comment:
Any particular reason to pass unique_ptr refs instead of plain SVGResources& refs?

Reply:
Because I am not that familiar with unique_ptr, but I think I got it with the latest cahnge.